src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,7 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        System.out.println(lottery.returnBall());
```

Comment:
The method `returnBall()` should be renamed to `getRandomBall()` to align with the task requirements. Also, ensure that the main method creates and prints three balls, not just one.